src/btf.cpp

```diff
@@ -714,6 +714,40 @@ std::set<std::string> BTF::get_all_structs() const
   return structs;
 }
 
+std::unordered_set<std::string> BTF::get_all_iters_from_btf(
+    const struct btf *btf) const
+{
+  std::unordered_set<std::string> iter_set;
+  const std::string prefix = "bpf_iter__";
+  __s32 id, max = (__s32)type_cnt(btf);
+  for (id = start_id(btf); id <= max; id++)
+  {
+    const struct btf_type *t = btf__type_by_id(btf, id);
+
+    if (!t || !(btf_is_struct(t)))
+      continue;
+
+    const std::string name = btf_str(btf, t->name_off);
+    if (name.size() >= prefix.size() &&
```

Comment:
Although it shouldn't occur in BTF, unnamed iterator is not correct, right?
```suggestion
    if (name.size() > prefix.size() &&
```